frontend/components/AddressForm/index.jsx

```diff
@@ -16,26 +16,24 @@ import {
   NAVIGATOR_SAVE_BUTTON_ENABLE,
   NAVIGATOR_SAVE_BUTTON_DISABLE,
 } from '../../constants/EventTypes';
-import config from '../../config';
 import connect from './connector';
-import countries from './countries';
+import iso3166 from '../../common/iso-3166-2';
 import style from './style';
 
-const { splitDefaultAddressesByTags = [], addressFields = [] } = config;
-
 /**
+ * @param {string[]} countryCodes from config
  * @return {Object}
  */
-const countriesList = () => Object.keys(countries).reduce((reducer, countryCode) => ({
+const countriesList = countryCodes => countryCodes.reduce((reducer, countryCode) => ({
   ...reducer,
-  [countryCode]: countries[countryCode].name,
+  [countryCode]: iso3166[countryCode].name,
 }), {});
 
 /**
  * @param {string} countryCode country
  * @return {Object}
  */
-const provincesList = countryCode => countries[countryCode].divisions;
+const provincesList = countryCode => iso3166[countryCode].divisions;
```

Comment:
The province list was moved to the `FormBuilder` component. This will most likely cause conflicts and we should merge it with `PWA-852`, before we merge it into `PWA-706-address-management`.